testing/integration/upgrade_integrations_server_test.go

```diff
@@ -0,0 +1,140 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+//go:build integration
```

Comment:
this no longer belongs in the main `testing/integration` package. Ideally there should be a dedicated `fips` one but until that day this should live inside `testing/integration/ess`